tests/external/iree-test-suites/onnx_models/onnx_models_gpu_rocm_rdna3.json

```diff
@@ -1,7 +1,7 @@
 {
   "config_name": "gpu_rocm_rdna3",
   "iree_compile_flags": [
-    "--iree-hal-target-backends=rocm",
+    "--iree-hal-target-device=hip",
     "--iree-hip-target=gfx1100"
   ],
```

Comment:
We might want to rename these configs to gpu_hip_rdna3 (config_name here, file name, workflow job name) now that the target _device_ is set to "hip".

Reply:
happy to do it - what does that entail? will that mess anything up if I just rename the file and fix refs in the repo? (not sure if anything external is hardcoded to these)